### ISSUE TITLE
Unable to use 'ord wallet receive' after creating wallet

### ISSUE THREAD

**Issue by 77656233** [ordinals/ord#1649]
```
root@sd-id:~# ord wallet create
error: JSON-RPC error: RPC error response: RpcError { code: -4, message: "Wallet file verification failed. Failed to create database path '/root/.bitcoin/ord'. Database already exists.", data: None }
root@sd-id:~# ord wallet receive
error: wallet "ord" contains unexpected output descriptors, and does not appear to be an `ord` wallet, create a new wallet with `ord wallet create`
root@sd-is:~#
```

Any idea how to solve the problem? I already deleted the wallet and created a new wallet but always get the same result. Using 0.4.2 compiled verwion.

**Comment by ChristianGrieger** [comment]
got it solved by create and use wallet with other name as ord

**Comment by cuamckuu** [comment]
> got it solved by create and use wallet with other name instead of using the standard created ord

Can you provide instruction on changing name? I can't see any name related options on `ord wallet create`

**Comment by cuamckuu** [comment]
Found it

```bash
ord --wallet test wallet create
ord --wallet test wallet receive
```

**Comment by ChristianGrieger** [comment]
Oh sorry just saw the message

**Comment by raphjaph** [comment]
Closing since this seems to be solved

**Comment by cuamckuu** [comment]
This is a bug, and it's not solved as default wallet name isn't working

**Comment by raphjaph** [comment]
Did you use any other version of `ord`? Did this happen on first use?

**Comment by torkelrogstad** [comment]
I'm also seeing this, with version 0.5.1. Not able to replicate it now, but happened after opening and closing Bitcoin Core (both QT and terminal) a few times, after having created an ord wallet.

**Comment by lepar** [comment]
Update:
The wallet that I created before that was working, is not working now when I try to generate address "ord1 wallet receive".
Maybe error is not in the generating but in the receive.
Versions 0.5.0 and 0.5.1 same error

**Comment by lepar** [comment]
```
lepar@lepar:~/Desktop$ ord --bitcoin-data-dir /media/lepar/nick/bitcoin-core wallet create
{
  "mnemonic": "",
  "passphrase": ""
}
lepar@lepar:~/Desktop$ ord --bitcoin-data-dir /media/lepar/nick/bitcoin-core wallet receive
error: wallet "ord" contains unexpected output descriptors, and does not appear to be an `ord` wallet, create a new wallet with `ord wallet create`

```

Getting this error right after creating the wallet. Not sure if it has to do anything with cache or something

**Comment by kvnn** [comment]
I'm seeing this on Ubuntu after (1) creating a wallet via python's `subprocess` and (2)  removing the `wallet.dat` file (with `rm`)

**Comment by estevanpedro** [comment]
I fixed. I had created a wallet using `bitcoin-cli` and later on deleted all of them in `.bitcoin/wallets`, `ord` wallet as well. After all this mess, i tried to create a new wallet using `ord wallet create`, but this `error: wallet "ord" contains unexpected output descriptors` was still there. So i did unload the wallets and deleted the folder using `rm -r .bitcoin/wallets`. Created a new wallet using `ord wallet create` and it solved.
